lib/ui/painting.dart

```diff
@@ -5366,12 +5366,26 @@ abstract class Canvas {
   void drawPath(Path path, Paint paint);
 
   /// Draws the given [Image] into the canvas with its top-left corner at the
-  /// given [Offset]. The image is composited into the canvas using the given [Paint].
+  /// given [Offset].
+  ///
+  /// The image is composited into the canvas using the given [Paint], by
+  /// applying the [Paint.color] (alpha channel only), [Paint.blendMode],
+  /// [Paint.colorFilter], [Paint.maskFilter], and [Paint.imageFilter]
+  /// properties of the `paint` argument (in that order).
+  ///
+  /// Multiple calls to this method with different arguments (from the same
+  /// image) can be batched into a single call to [drawAtlas] to improve
```

Comment:
Not sure I follow this part. The additional color + image blending and transforms can be different per draw, but otherwise all draws share the same filters/alpha.

Reply:
yeah maybe this isn't worth saying for drawImage. I'll remove it.